accounts/keystore/account_cache_test.go

```diff
@@ -350,7 +350,7 @@ func TestUpdatedKeyfileContents(t *testing.T) {
 		return
 	}
 	// needed so that modTime of `file` is different to its current value after forceCopyFile
-	time.Sleep(time.Second)
+	os.Chtimes(file, time.Now().Add(-time.Second), time.Now().Add(-time.Second))
```

Comment:
curious -- when I test locally, I don't even need these. I can just comment out the Sleep and not even replace it, and the test still passes. Not sure how/why, or if this is needed on other platforms ? 

I guess the mtime on linux has a very high resolution. 

Some googling says that "Windows file times have a resolution of, at best, 100-nanoseconds)"

Reply:
> File timestamps precision is limited to one second for EXT3, one microsecond for UFS, and one nanosecond for EXT4 (at least according to experience).

https://unix.stackexchange.com/questions/11599/determine-file-system-timestamp-precision